### DISCUSSION TITLE
Gimli Bootstrap: The smart DevTools extension for Bootstrap developers.

### DISCUSSION THREAD

**Discussion by JHjertvik** [twbs/bootstrap#39869] [Show & Tell]
[Link to extension](https://chromewebstore.google.com/detail/gimli-bootstrap/oeohjgdiapbcnlifpfhmppjkdkabkfbj)

[Short video introduction](https://www.youtube.com/watch?v=xI05jhoqoPw)

Feature Highlights 🔥

✔️ Works on any website.

✔️ Autocomplete:
Intelligent suggestions for class names. It also supports searching for CSS property names.

✔️ Instant feedback: Preview changes on hover.

✔️ Quickly toggle utility values with Cmd/Ctrl + Click.

✔️ Intuitive UI

![splashBSPH](https://github.com/twbs/bootstrap/assets/49238302/7ee640ff-b0d7-4ec8-b9bb-f5794a6eaa26)

**Comment by bacloud22** [comment]
This is marvelous for anyone with limited CSS knowledge ! Thank you!
